### PR TITLE
Kusama styling

### PR DESCRIPTION
**Pull request by erler**
Collection of previous smaller styling PRs. This is not complete, but would like to merge to keep this PR from growing.

### CODE REVIEW

File: src/components/account/AccountSelector.js [hunk 1]
```diff
@@ -41,16 +41,16 @@ const AccountToggleItem = ({ account, balance }) => {
...
     ContentElement = (
-      <div style={{ marginLeft: caretSize + caretMargin, fontSize: '1.25rem' }}>
+      <div style={{ marginLeft: caretSize + caretMargin }}>
         Select Account
       </div>
     );
   }
   return (
     <>
-      <div className="account-toggle-item rounded border border-primary d-flex flex-row align-items-center p-2">
```
Comment (hamidra) [inline src/components/account/AccountSelector.js:51]: Are these border styles kusama specific or do we want to remove them for polkadot as well?! I think we should plan to eventually have a single repo/code for both apps with some theme config and should be able to deploy each app simply by importing the correct .scss theme (static import per app is fine IMO). So if these border styles are needed for polkadot we want to add them to polkadot.scss.

File: src/components/CardHeader.js [hunk 2]
```diff
@@ -14,19 +14,13 @@ export default function CardHeader({ cardText, title, backClickHandler }) {
           style={{
             ...(backClickHandler
               ? { paddingLeft: colPaddingX, paddingRight: colPaddingX }
-              : {}),
+              : {})
           }}>
           {backClickHandler && (
             <ArrowLeft
+              className='back-arrow'
```
Comment (hamidra) [inline src/components/CardHeader.js:21]: We need to add this class for polkadot.scss as well if we are removing the style

**Comment by hamidra** [comment]
Looks very nice. Just I think we should plan to make both style variants consistent since we want to merge the the codebases eventually after the campaign.

**Comment by erler** [comment]
Apologies, I made a git mess, which made this un-reviewable. Started fresh branch from my last clean commit. PR [here](https://github.com/hamidra/dotdrop/pull/76)

**Pull request closed without merging**